ray-operator/hack/local_deploy.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+PROJECT_ROOT="${SCRIPT_DIR}/../.." # hack/ -> ray-operator/ -> kuberay/
+
+# Path to your Helm chart, relative to the PROJECT_ROOT (kuberay/)
+HELM_CHART_RELATIVE_PATH="helm-chart/kuberay-operator"
+HELM_CHART_PATH="${PROJECT_ROOT}/${HELM_CHART_RELATIVE_PATH}"
+
+RAY_OPERATOR_PATH="${PROJECT_ROOT}/ray-operator"
+
+# --- Configuration Variables ---
+IMAGE_TAG="kuberay-dev-tag"
+KIND_CLUSTER_NAME="kuberay-dev"
+KIND_NODE_IMAGE="kindest/node:v1.24.0"
+HELM_RELEASE_NAME="kuberay-operator"
+
+delete_kind_cluster() {
+  echo "--- Deleting Kind Cluster ---"
+  if kind get clusters | grep -q "${KIND_CLUSTER_NAME}"; then
+    echo "Kind cluster '${KIND_CLUSTER_NAME}' found. Deleting it..."
+    kind delete cluster --name "${KIND_CLUSTER_NAME}"
+  else
+    echo "Kind cluster '${KIND_CLUSTER_NAME}' not found. Skipping deletion."
+  fi
+}
+
+create_kind_cluster() {
+  echo "--- Creating Kind Cluster ---"
+  echo "Creating Kind cluster '${KIND_CLUSTER_NAME}' with image '${KIND_NODE_IMAGE}'..."
+  kind create cluster --name "${KIND_CLUSTER_NAME}" --image="${KIND_NODE_IMAGE}"
+  echo "Kind cluster nodes are ready."
+}
+
+# Parsing for arguements
+SHOW_LOGS=false 
+while [[ "$#" -gt 0 ]]; do
+  case "$1" in
+    -l|--logs)
+      SHOW_LOGS=true
+      shift # Remove --logs or -l from processing
+      ;;
+    *)
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
+
+# Step 1: Ensure a clean Kind cluster
```

Comment:
I would remove this comment, the log message below already explains it. Also we're missing step 2 and 3